schemas/dto/actionStatus.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Enum of action result statuses",
+  "javaType": "org.folio.rest.jaxrs.model.ActionStatus",
+  "type": "string",
+  "additionalProperties": false,
+  "enum": [
+    "CREATED",
+    "UPDATED",
+    "MULTIPLE",
```

Comment:
Multiple_match? like a specific error status?

Reply:
no, `MULTIPLE` means multiple entities of the same type were affected during the processing of one mar record. For instance, some existing holdings record was updated by incoming record and 1 new holdings record was created by the same incoming record

Reply:
OK, thanks)